lib/store/chat.dart

```diff
@@ -460,8 +526,10 @@ class ChatRepository extends DisposableInterface
       if (id == monolog?.chat.value.id) {
         _monologShouldBeHidden = false;
         _chats[id] = await _putEntry(monolog!);
+        _paginationChats[id] = _chats[id]!;
```

Comment:
Давайте вместо `!` будем

```suggestion
        final ... entry = await _putEntry(monolog!);
        _chats[id] = entry;
        _paginationChats[id] = entry;
```

?